reflex-common/reflex-common/src/main/python/mcenter_cli/parallelm/mcenter_cli/main.py

```diff
@@ -167,6 +169,13 @@ def _model_upload(mclient, args):
         model_upload_parser.add_argument('name', action='store', help="Model name to use")
         model_upload_parser.add_argument('format', action='store', help="Model format to use")
 
+    def _add_action_log_commands(self):
+        action_log_parser = self._register_command('get-action-log', help='Get Action log for MCenter', action=list_actions)
+        action_log_parser.add_argument('--object', action='store', help='Objects for which to get actions')
+        action_log_parser.add_argument('--action', action='store', help='actions to query')
+        action_log_parser.add_argument('--username', action='store', help='User, whose actions should be queried')
+        action_log_parser.add_argument('--status', action='store', help='Show actions with particular status, [both, success, failed]')
```

Comment:
parser.add_argument('move', choices=['rock', 'paper', 'scissors'])
thats what you need.
Google for: python option choices

Reply:
Fixed